### PR TITLE
Fix pgvector support in @edgedb/generate

### PR DESCRIPTION
**Pull request by jaclarke**

### CODE REVIEW

File: packages/generate/test/pgvector.test.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        }[]
+      >
+    >(true);
+  });
+
+  test("params", async () => {
+    const query = e.params({ vec: e.ext.pgvector.vector }, ($) => e.select($));
+
+    const args = {
+      vec: Float32Array.from(
```
Comment (scotttrinh) [inline packages/generate/test/pgvector.test.ts:53]: Should we also test `number[]`? Maybe test that selecting either is deep equal?

File: packages/generate/test/testRunner.ts [hunk 2]
```diff
@@ -26,11 +29,37 @@ import {
...
+    });
+    console.log(`\nRunning tests...`);
+    await runCommand(
+      "yarn",
+      [
+        "test:ts",
+        ...(version.major < 3
+          ? [
+              `--testPathIgnorePatterns="${[
+                "pgvector",
```
Comment (scotttrinh) [inline packages/generate/test/testRunner.ts:54]: Love this! We might even be able to throw flags into the test name to get this less specific, but this works great.

**Review by scotttrinh — APPROVED** [review]
Great! No blockers.

File: packages/generate/test/pgvector.test.ts (hunk 1 above)
Comment (jaclarke) [reply packages/generate/test/pgvector.test.ts]: Yes, added

**Pull request merged**